deps/lua/src/Makefile

```diff
@@ -8,7 +8,7 @@
 PLAT= none
 
 CC= gcc
-CFLAGS= -O2 -Wall -fPIC $(MYCFLAGS)
+CFLAGS= -O2 -Wall $(MYCFLAGS) -fPIC
```

Comment:
remove -fPIC here because that it's already selected for the platform in MYCFLAGS